adoc/chapters/programming_interface.adoc

```diff
@@ -4704,295 +4704,358 @@ execution.
 The SYCL [code]#event# class provides the common reference semantics (see
 <<sec:reference-semantics>>).
 
-The constructors and member functions of the SYCL [code]#event# class are listed
-in <<table.constructors.event>> and <<table.members.event>>, respectively.
-The additional common special member functions and common member functions are
-listed in <<table.specialmembers.common.reference>> and
-<<table.hiddenfriends.common.reference>>, respectively.
-
 // Interface for class: event.h
 [source,,linenums]
 ----
 include::{header_dir}/event.h[lines=4..-1]
 ----
 
+[[sec:event-ctors]]
+==== Constructors
 
-[[table.constructors.event]]
-.Constructors of the [code]#event# class
-[width="100%",options="header",separator="@",cols="65%,35%"]
-|====
-@ Constructor @ Description
-a@
-[source]
+.[apititle]#Default constructor#
+[source,role=synopsis,id=api:event-ctor]
 ----
 event()
 ----
-   a@ Constructs an [code]#event# that is immediately ready.  The
-      [code]#event# has no dependencies and no associated commands.
-      Waiting on this [code]#event# will return immediately and querying
-      its status will return [code]#info::event_command_status::complete#.
 
-The event is constructed as though it was created from a default-constructed
-[code]#queue#.  Therefore, its backend is the same as the backend from the
-default device.
+_Effects:_ Constructs an [code]#event# that is immediately ready.
+The [code]#event# has no dependencies and no associated commands.
+Waiting on this [code]#event# will return immediately and querying its status
+will return [api]#info::event_command_status::complete#.
 
-|====
+_Remarks:_ The event is constructed as though it were created from a
+default-constructed [code]#queue#.
+Therefore, its backend is the same as the backend of the device selected by
+[code]#default_selector_v#.
```

Comment:
I've added the reference to `default_selector_v`, because SYCL doesn't actually have a default device.